pkg/volume/emptydir/empty_dir.go

```diff
@@ -271,8 +316,14 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 		return nil
 	}
 
+	var options []string
+	// Linux system default is 50% of capacity.
+	if ed.sizeLimit != nil && ed.sizeLimit.Value() > 0 {
+		options = []string{fmt.Sprintf("size=%d", ed.sizeLimit.Value())}
+	}
```

Comment:
is it a problem if `EmptyDir.SizeLimit` is greater than calculated size limit but pod does not get evicted until `EmptyDir.SizeLimit` in API? should eviction thresholds be updated as well?

Reply:
the limit is distinct from usage.

the limit will not be able to exceed allocatable memory for a pod.

Reply:
so to answer, i do not think there is a problem here.